pkg/cluster/cluster.go

```diff
@@ -479,6 +479,15 @@ func (c *Cluster) pollPods() ([]*v1.Pod, []*v1.Pod, error) {
 }
 
 func (c *Cluster) updateStatus() error {
+	if c.bm != nil {
+		bs, err := c.bm.getStatus()
```

Comment:
ok. interesting. this is possible indeed. also 30s is too long anyway. lets shorten the timeout.